core_impl/src/main/java/io/opencensus/implcore/tags/TagsComponentImplBase.java

```diff
@@ -35,4 +41,15 @@ public Tagger getTagger() {
   public TagPropagationComponent getTagPropagationComponent() {
     return tagPropagationComponent;
   }
+
+  @Override
+  public TaggingState getState() {
+    return state.get();
+  }
+
+  @Override
+  public void setState(TaggingState newState) {
+    state.set(checkNotNull(newState));
```

Comment:
it is better to use checkNotNull(newState, "newState") because the message will tell you what variable was null. gRPC people told me to do that because will help with debugging.

Reply:
Done